hawkular-alerts-bus-api/src/main/java/org/hawkular/alerts/bus/api/MetricDataMessage.java

```diff
@@ -86,6 +86,8 @@ public String toString() {
      */

```

Comment:
It might be nice if Alerting and Metrics could share classes for this messaging.  I don't think metrics makes the message classes public at the moment...